frontend/src/screens/studyBanner.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useCallback, useState, useEffect } from 'react';
+import { styled } from '@common';
+import { colors } from '@theme';
+import { useSearchStudies, useParticipantStudies } from './learner/studies';
+import { IconArrowRight } from '@tabler/icons-react';
+import { useNavigate } from 'react-router-dom';
+
+const AchievementsContainer = styled.div({
```

Comment:
We want to move away from styled components

Reply:
I would rewrite this using mantine components and avoid using styled components in the future 🙏